### PR TITLE
[move-prover] Added the local specs of libra_account.move

### PR DESCRIPTION
**Pull request by junkil-park**
- Added the specifications for 21 functions (out of 40) in the LibraAccount module
    - Event-related functions
        - fresh_guid, new_event_handle_impl, new_event_handle, emit_event, write_to_event_store, destroy_handle event
    - Functions with simple call-dependencies
        - extract_sender_withdrawal_capability, restore_withdrawal_capability, rotate_authentication_key_for_account, rotate_authentication_key, rotate_authentication_key_with_capability, extract_sender_key_rotation_capability, restore_key_rotation_capability, sequence_number_for_account, sequence_number, authentication_key, delegated_key_rotation_capability, delegated_withdrawal_capability, withdrawal_capability_address, key_rotation_capability_address, exists
- Added the test case file (`verify_lcs.move`) to verify the native function of the LCS module (i.e., `to_bytes`)
- Updated the builtin boogle model for `$LibraAccount_write_to_event_store`
- Added the native spec function `serialize` to the LCS module
- Added the auxiliary spec functions to the Vector module, and 
- Modified `bytecode_translator.rs` to enable translating them to Boogie

## Motivation

To verify LibraAccount module

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

`cargo test`

In `libra/language/move-prover/tests/sources/stdlib/modules`, run the following command:
```
cargo run -- --only-verify-spec libra_account.move hash.move lbr.move lcs.move libra.move libra_transaction_timeout.move transaction.move vector.move libra_time.move
```

### CODE REVIEW

**Comment by junkil-park** [comment]
@cbarrettfb FYI

File: language/move-prover/src/prelude.bpl [hunk 1]
```diff
@@ -1033,6 +1033,9 @@ axiom (forall ta1, ta2: TypeValue ::
        (forall v1, v2: Value :: IsEqual($serialize(ta1, v1), $serialize(ta2, v2))
            ==> IsEqual(v1, v2) && (ta1 == ta2)));
 
+function $LCS_serialize(ta: TypeValue, v: Value): Value {
```
Comment (DavidLDill) [inline language/move-prover/src/prelude.bpl:1036]: It looks like we could rename "$serialize" everywhere to "$LCS_serialize" and save this redundant function definition.

Comment (junkil-park) [reply language/move-prover/src/prelude.bpl]: Done renaming.

**Pull request closed without merging**